src/neph/program.cr

```diff
@@ -9,6 +9,13 @@ class Program
     opt_parser.banner = "Usage: neph [options] [job name]"
     opt_parser.on "-v", "--version", "Show software version" { STDERR.puts VERSION; exit 0 }
     opt_parser.on "-h", "--help", "Show this help message" { STDERR.puts opt_parser; exit 0 }
+    opt_parser.on "-f", "--file", "Specify an alternative file" do |config|
```

Comment:
`config` is not a good variable name. It should be meaniingful, e.g. `filename` or `alternative_file`.